test/token/ERC20/utils/SafeERC20.test.js

```diff
@@ -192,8 +230,17 @@ function shouldRevertOnAllCalls(reason) {
     // [TODO] make sure it's reverting for the right reason
     await expectRevert.unspecified(this.mock.$safeDecreaseAllowance(this.token.address, constants.ZERO_ADDRESS, 0));
   });
+
+  it('reverts on forceApprove', async function () {
+    // [TODO] make sure it's reverting for the right reason
```

Comment:
Do you plan to address this TODO in this PR?

Reply:
This TODO was copied from the other TODO in this file. 

I'll rework the test file completelly